packages/common/src/EthersProviderUtils.ts

```diff
@@ -0,0 +1,106 @@
+import { ethers } from "ethers";
```

Comment:
How similar is this to the Across retry provider? Can you point out any changes you made, so we can review those more carefully?

Reply:
Its basically a stripped down version as can be seen from the last revision in [this gist](https://gist.github.com/Reinis-FRP/387b72a4daac140e22c1235df7b1686b/revisions?diff=unified). Most notable changes are as follows:

- All rate limiting functionality is removed, thus `EthersRetryProvider` does not extend `RateLimitedProvider`, but instead is directly extending  ethers `StaticJsonRpcProvider`.
- Quorum and result comparison functionality is removed. Nodes are processed in the order and the first successful result is returned.
- Verification of constructor parameters is moved out to the `getRetryProvider` method (named `getProvider` in Across) that is the only method how protocol applications would create redundant node provider.
- There is also no storage of failing node errors and error is thrown only if all nodes failed to respond.